exercises/high-scores/canonical-data.json

```
@@ -0,0 +1,107 @@
+{
+  "exercise": "high-scores",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Returns the list of scores",
+      "property": "scores",
+      "input": {
+        "scores": [30, 50, 20, 70]
+      },
+      "expected": [30, 50, 20, 70]
+    },
+    {
+      "description": "Returns the last score",
+      "property": "latest",
+      "input": {
+        "scores": [100, 0, 90, 30]
+      },
+      "expected": 30
+    },
+    {
+      "description": "Returns the highest score",
+      "property": "highest",
+      "input": {
+        "scores": [40, 100, 70]
+      },
+      "expected": 100
+    },
+    {
+      "description": "Returns the top 3 scores",
+      "cases": [
+        {
+          "description": "Returns personal bests",
+          "property": "top",
+          "input": {
+            "scores": [50, 30, 10]
+          },
+          "expected": [50, 30, 10]
+        },
+        {
+          "description": "Returns personal bests highest to lowest",
+          "property": "top",
+          "input": {
+            "scores": [20, 10, 30]
+          },
+          "expected": [30, 20, 10]
+        },
+        {
+          "description": "Returns personal bests when there is a tie",
+          "property": "top",
+          "input": {
+            "scores": [40, 20, 40, 30]
+          },
+          "expected": [40, 40, 30]
+        },
+        {
+          "description": "Returns personal bests when there are less than 3",
+          "property": "top",
+          "input": {
+            "scores": [30, 70]
+          },
+          "expected": [70, 30]
+        },
+        {
+          "description": "Returns personal bests when there is only one",
+          "property": "top",
+          "input": {
+            "scores": [40]
+          },
+          "expected": [40]
+        },
+        {
+          "description": "Returns three top scores from a long list",
+          "property": "top",
+          "input": {
+            "scores": [10, 30, 90, 30, 100, 20, 10, 0, 30, 40, 40, 70, 70]
+          },
+          "expected": [100, 90, 70]
+        }
+      ]
+    },
+    {
+      "description": "Returns report when highest is latest",
+      "property": "report",
+      "input": {
+        "scores": [20, 40, 0, 30, 70]
+      },
+      "expected": "Your latest score was 70. That's your personal best!"
+    },
+    {
+      "description": "Returns report when there is a tie",
+      "property": "report",
+      "input": {
+        "scores": [20, 40, 70, 30, 70]
+      },
+      "expected": "Your latest score was 70. That's your personal best!"
+    },
+    {
+      "description": "Returns report when not highest score",
```

To me, the description could maybe be a bit clearer:

```suggestion
      "description": "Returns report when highest is not latest"",
```

---

Thanks for the suggestion!